numpy/core/src/umath/ufunc_object.c

```diff
@@ -1974,6 +1965,40 @@ _check_ufunc_fperr(int errmask, PyObject *extobj, const char *ufunc_name) {
     return ret;
 }
 
+/*
+ * Returns a new reference
+ * TODO: store a reference in the ufunc object itself, rather than 
+ *       constructing one each time
+ */
+static PyObject *
+_get_identity(PyUFuncObject *ufunc, npy_bool *reorderable) {
+    switch(ufunc->identity) {
+    case PyUFunc_One:
+        *reorderable = 1;
+        return PyInt_FromLong(1);
```

Comment:
This used to be `True` when used internally, and `1` when viewed externally.

The upshot is that this fixes #8860.

Ideally, we'd use `True` for functions like `logical_or`, and `1` for functions like `add`